cf-servicesinfo.rb

```diff
@@ -10,38 +10,22 @@
 
 
 def exempt_space(space_name)
```

Comment:
can we change this to `exempt_space?` or `is_exempt_space/is_exempt_space?`. just so it's easier to know that it's meant to be returning true or false